kratos/includes/define.h

```diff
@@ -803,6 +803,7 @@ namespace Kratos
 //Print Trace if defined
 #define KRATOS_WATCH(variable) std::cout << #variable << " : " << variable << std::endl;
 #define KRATOS_WATCH_CERR(variable) std::cerr << #variable << " : " << variable << std::endl;
+#define KRATOS_WATCH_MPI(variable, mpi_data_comm) std::cout << "RANK " << mpi_data_comm.Rank() << "/" << mpi_data_comm.Size() - 1 << "\t"; KRATOS_WATCH(variable);
```

Comment:
```suggestion
#define KRATOS_WATCH_MPI(variable, mpi_data_comm) std::cout << "RANK " << mpi_data_comm.Rank() << "/" << mpi_data_comm.Size()  << "    "; KRATOS_WATCH(variable);
```

please follow the standard conventions of MPI

otherwise if you run with one rank it will print 0 :)

also I would not use tab as it can be quite long on some terminals